src/placetobe-app/placetobe-campagnes-page.html

```diff
@@ -0,0 +1,109 @@
+<!--
+A routing element for all campaign(s) related pages.
+
+Example:
+
+    <placetobe-campagnes-page
+    campaign-route="{{subRoute}}"
+    ></placetobe-campagnes-page>
+-->
+
+<link rel="import" href="../../bower_components/polymer/polymer.html">
+<link rel="import" href="../../bower_components/placetobe-route/placetobe-route.html">
+<link rel="import" href="../../bower_components/placetobe-campaign-page/placetobe-campaign-page.html">
+<link rel="import" href="placetobe-landing-page-styles.html">
+<link rel="import" href="placetobe-investment-failed.html">
+
+
+<dom-module id="placetobe-campagnes-page">
+  <template>
+    <style>
+      :host {
+        display: block;
+      }
+
+      :host h1 {
+        margin-top: calc( var(--placetobe-margin) * 8 );
+      }
+
+      [hidden] {
+        display: none;
+      }
+    </style>
+
+    <!-- obtaines campaign(s) route from parent and exposes information about campaign(s) (for example campaignId).-->
+    <!--TODO: pattern :campaignId can also be /all so name not optimal -->
+    <placetobe-route
+      route="{{campaignRoute}}"
+      pattern="/:campaignId"
+      data="{{routeData}}"
+      tail="{{subRoute}}"
+    ></placetobe-route>
+
+    <!-- Gathers campaign(s) specific information for remaining route, for example post invest status -->
+    <placetobe-route
+      route="{{subRoute}}"
+      pattern="/:status"
+      data="{{subRouteData}}"
+    ></placetobe-route>
+
+    <!-- If routeData indicates that the page is about one campaign and if there is no subRouteData (for example post invest)-->
+    <template is="dom-if" if="{{!subRouteData.status}}">
+      <placetobe-campaign-page api-endpoint="{{apiEndpoint}}" campaign-id="{{routeData.campaignId}}"></placetobe-campaign-page>
+    </template>
+    
+    <!-- If post invest was unsuccesful (route: /campagnes/:campaignId/reject)  -->
+    <template is="dom-if" if="{{_sameString(subRouteData.status, 'reject', 'cancel', 'error')}}">
```

Comment:
@SPrinss This method is too complex and abstract now. I'd suggest creating an `investmentSuccessfull` computed property `boolean`, to handle both this and successes on line 61